### PR TITLE
Extra sensors

### PR DESCRIPTION
**Pull request by benjamin-dcs**

### CODE REVIEW

File: custom_components/ithodaalderop/definitions.py [hunk 1]
```diff
@@ -181,14 +181,55 @@ class IthoBinarySensorEntityDescription(BinarySensorEntityDescription):
...
+    IthoSensorEntityDescription(
+        json_field="Highest RH concentration (%)",
+        key=MQTT_STATETOPIC["hru"],
+        translation_key="highest_received_rh_value",
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
     IthoSensorEntityDescription(
         json_field="hum",
```
Comment (benjamin-dcs) [inline custom_components/ithodaalderop/definitions.py:234]: @jasperslits I highly doubt this json_field is correct. Can you verify this somehow?

Comment (benjamin-dcs) [reply custom_components/ithodaalderop/definitions.py:234]: Same for "temp"

**Comment by jasperslits** [comment]
Perhaps for CVE it makes sense to change the state topic from 'hru' to 'cve'? Not directly related to this PR

So we'd have:

MQTT_STATETOPIC = {
    "hru": "ithohru/ithostatus",
    **"cve": "ithocve/ithostatus",**
    "wpu": "ithowpu/ithostatus",
    "remotes": "ithohru/remotesinfo",
    "autotemp": "ithotemp/ithostatus",
}

**Comment by benjamin-dcs** [comment]
> "remotes": "ithohru/remotesinfo",

Will become troublesome for `"remotes": "ithohru/remotesinfo",` I think. Both CVE and HRU can have remotes. Better make it something like `itho` or `itho-wifi-addon/` or .. for all?

**Comment by benjamin-dcs** [comment]
Should we account for multiple WiFi addons?

**Comment by jasperslits** [comment]
Didnt think of the remotes but you're right. I like to have a meaningful name for both HRU and CVE combined instead of just "itho"

Maybe we should have a poll. 

I have 3 wifi add-on hence the prefix. One for HRU, one for WPU and one for autotemp

**Comment by benjamin-dcs** [comment]
We can determine the topic for the remotes based on the config of the integration instance

**Comment by benjamin-dcs** [comment]
Continue here for topic discussion:
https://github.com/jasperslits/haithowifi/pull/34

File: custom_components/ithodaalderop/definitions.py [hunk 2]
```diff
@@ -181,14 +181,55 @@ class IthoBinarySensorEntityDescription(BinarySensorEntityDescription):
...
+        translation_key="filter_dirty",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        icon_off="mdi:air-filter",
+        icon_on="mdi:vacuum-outline",
+    ),
+)
+
 CVESENSORS: tuple[IthoSensorEntityDescription, ...] = (
+    IthoSensorEntityDescription(
+        json_field="Error",
```
[the remark this answers is not in the document]
Comment (benjamin-dcs) [reply custom_components/ithodaalderop/definitions.py:197]: At least you know there's 'something' wrong. Probably you can check on the unit for flashing leds. I'll check the Itho service tool later for codes

File: custom_components/ithodaalderop/definitions.py (hunk 1 above)
Comment (benjamin-dcs) [reply custom_components/ithodaalderop/definitions.py:234]: Thanks. The naming of these is just so different compared to the rest

**Comment by jasperslits** [comment]
Perhaps this can be rebased on main and then merged?

**Comment by benjamin-dcs** [comment]
Will update this branch/PR a.s.a.p and look into the `mqtt-examples` for `autotemp` and `wpu` for any useful sensor for those units

**Comment by benjamin-dcs** [comment]
Not ready yet!

**Pull request merged**